README.md

```diff
@@ -1,2 +1,53 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # capstone-1
 Building an online website for a conference
+
+![screenshot](./images/2021-10-12.png)
+
+.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+## Application Page url
+
+
+## Getting Started
+
+> To get a local copy follow the following steps:
+
+- Copy this link https://github.com/phelian23/capstone-1.git
+- Open your terminal or command line
+- Run git clone and paste the link
+- Open the folder with your code editor
+- Create a branch to work on
+
+## Authors
+
+👤 **Phelian23**
+
+- GitHub: [@phelian23](https://github.com/phelian23)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
```

Comment:
6. The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/). (Please add this information in order to comply with the project requirements)